### PR TITLE
Fix typo: s/pagable/pageable/

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
Pagable and pageable are both equally (in)correct. At least last time I googled it.

**Comment by jforissier** [comment]
Really?
When I type "pagable" in Google, I get: "Did you mean: pageable".
Also, thefreedictionary.com has a definition for "pageable memory" but has nothing for "pagable" (in English): http://encyclopedia2.thefreedictionary.com/pageable+memory
And, Linux has 6 instances of 'pageable', zero of 'pagable'.
```
jerome@jfw540:~/work/linux (master)$ git grep pagable | wc -l
0
jerome@jfw540:~/work/linux (master)$ git grep pageable | wc -l
6
```

**Comment by jbech-linaro** [comment]
I asked Mark G about it (native American) and he believed `pageable` is the correct way of writing it.
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: scripts/gen_hashed_bin.py [hunk 1]
```diff
@@ -125,16 +125,16 @@ def main():
 	args = get_args()
 	init_bin_size 	  = args.init_size
 	tee_pager_fname   = args.tee_pager_bin
-	tee_pagable_fname = args.tee_pagable_bin
+	tee_pageable_fname = args.tee_pageable_bin
```
Comment (pascal-brand-st-dev) [inline scripts/gen_hashed_bin.py]: Align the "="

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jforissier** [comment]
Review comment addressed + rebased.

**Comment by jenswi-linaro** [comment]
I can't find pageable (or pagable) in my dictionary, but I've found a couple of other words which are similar, compare with reusable, removable, erasable etc. However, if you prefer it this way please go ahead.

**Comment by pascal-brand-st** [comment]
Manually merged.

**Pull request closed without merging**